Dockerfile

```diff
@@ -78,19 +90,30 @@ RUN --mount=target=/host \
         --nogpgcheck \
         builddep rpmbuild/SPECS/${PACKAGE}.spec
 
-# We use the "nocache" writable space to generate code where necessary, like
-# the variant-specific models.
+# We use the "nocache" writable space to generate code where necessary, like the variant-
+# specific models.
 USER builder
 RUN --mount=source=.cargo,target=/home/builder/.cargo \
     --mount=type=cache,target=/home/builder/.cache,from=cache,source=/cache \
-    --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/models/src/variant,from=variantcache,source=/variantcache \
-    --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/logdog/conf/current,from=variantcache,source=/variantcache \
+    --mount=type=cache,target=/home/builder/rpmbuild/BUILD/sources/models/src/variant,\
+from=variantcache,source=/variantcache \
```

Comment:
This looks really odd to me.  If the long-ish line really bugs you, I guess I'd suggest making the `target` equivalent to the `from` on its own new line..?